packages/sampler/stories/qa-stories/ExpandableInput.js

```diff
@@ -25,4 +25,37 @@ storiesOf('ExpandableInput', module)
 				type={text('type')}
 			/>
 		)
+	)
+	.add(
+		'Multiple ExpandableInputs',
+		() => (
+			<div>
+				<ExpandableInput
+					disabled={boolean('disabled-1', false)}
+					iconAfter={select('iconAfter-1', iconNames)}
+					iconBefore={select('iconBefore-1', iconNames)}
+					noneText={text('noneText-1', 'nothing inputted')}
+					onChange={action('onChange')}
+					onClose={action('onClose')}
+					onOpen={action('onOpen')}
+					open={boolean('open-1', true)}
```

Comment:
I don't think we want to make this component "controlled". I believe the main purpose of the sample is to test spotlight focus while it's opened/closed and so on.

Reply:
Well, that will make it a lot simpler :D